### PR TITLE
 'close' set to false on default in 'Chip'

### LINKED ISSUE #487: `Chip` to have default 'close' set to false in defaultProps
help-wanted

### PR DESCRIPTION
**Pull request by sanketsingh24**
# Description
 'close' set to false on default in 'Chip', fixes #487 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] I have not generated a new package version. (the maintainers will handle that)

### CODE REVIEW

File: src/Chip.js [hunk 1]
```diff
@@ -12,11 +12,11 @@ const Chip = ({ children, close }) => {
 
 Chip.propTypes = {
   children: PropTypes.node,
-  /**
-   * If show a close icon
```
Comment (alextrastero) [inline src/Chip.js]: I think we should keep the description (maybe a rewording)

Comment (alextrastero) [reply src/Chip.js]: `Show a close icon`

**Comment by alextrastero** [comment]
And thanks for contributing!!

File: src/Chip.js (hunk 1 above)
Comment (sanketsingh24) [reply src/Chip.js]: Ok, will add that

**Comment by alextrastero** [comment]
Smashing! Thanks

**Comment by sanketsingh24** [comment]
It was my pleasure.

**Pull request merged**